Dockerfile

```diff
@@ -27,5 +27,6 @@ COPY --from=builder ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 COPY predictionprophet_deployment ./predictionprophet_deployment
 
 ENV PYTHONPATH=/app
+ENV PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
```

Comment:
The suggestion from library was either this or using a version of protobuf lower than 3.20.x, however due to conflicts, we can do only this.

Reply:
What's the problem with our docker images that this fixes? The images don't have a c++ compiler?

Reply:
The error was

```
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your `protos`, some other possible workarounds are:
 1. Downgrade the `protobuf` package to 3.20.x or lower.
 2. Set `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` (but this will use pure-Python parsing and will be much slower).
```

Reply:
sorrryyyy if i'm being slow, could you give a bit more context. When do we see this error? I would have guessed in the `build-and-push-image` python CD step? looks like they're passing without error though.

Reply:
No no, this was during the runtime in Kubernetes. This errored out while importing chroma in Python.

Reply:
ohhh okay. thanks!